pgmoon/json.moon

```diff
@@ -0,0 +1,10 @@
+encode_json = (tbl) ->
+  json = require "cjson"
+  enc = json.encode tbl
+  "'#{(enc\gsub "'", "''")}'"
```

Comment:
you can see in the array module how it was done, a little hacky but at least it re-uses the function: https://github.com/leafo/pgmoon/blob/master/pgmoon/arrays.moon#L7